lib/irc.js

```diff
@@ -813,6 +845,7 @@ Client.prototype.disconnect = function(message, callback) {
     }
     self.conn.end();
 };
+
```

Comment:
Thank you for adding these newlines! I'm a little unsure about whether they should be in this pull req, though. Would it be too much trouble to put them in a separate PR?